R/yhatR.R

```diff
@@ -589,22 +589,160 @@ yhat.deploy <- function(model_name, packages=c(), confirm=TRUE) {
   }
 }
 
+#' Deploy a batch model to Yhat servers
+#'
+#' @param job_name
+#' @param confirm boolean indicating whether to prompt before deploying
+#' @keywords deploy
+#' @export
+#' @examples
+
```

Comment:
this might not work w/ vignettes. but we haven't gotten to this yet. i'll show you when we run the build and check commands.